elasticsearch_metrics/metrics.py

```diff
@@ -26,20 +28,21 @@ def __new__(mcls, name, bases, attrs):  # noqa: B902
         template = getattr(meta, "template", None)
         abstract = getattr(meta, "abstract", False)
 
+        app_label = getattr(meta, "app_label", None)
+        # Look for an application configuration to attach the model to.
+        app_config = apps.get_containing_app_config(module)
+        if app_label is None:
+            if app_config is None:
+                if not abstract:
+                    raise RuntimeError(
+                        "Metric class %s.%s doesn't declare an explicit "
```

Comment:
I moved this out of the `if` block below because `registry.register` needs the app label.

This change means that *all* Metric classes must either (1) be located within a Django app or (2) explicitly set `Meta.app_label`.